src/ui/library/BottomSheet.tsx

```diff
@@ -44,10 +65,21 @@ export function useBottomSheet() {
         enablePanDownToClose={true}
         animateOnMount={true}
         {...props}>
-        <BottomSheetView onLayout={handleContentLayout}>{children}</BottomSheetView>
+        <BottomSheetView onLayout={handleContentLayout}>
+          {children ? children : bottomSheetContent ? bottomSheetContent : noContent}
```

Comment:
You can use typescript to force the user to always pass a children or the content to openBottomSheet so we don't have to handle the `noContent` case, it would also prevent the developers to forget passing any.. We should never display `No content`